### DISCUSSION TITLE
copilot is not working-我的copilot无法使用了 [auth] Invalid copilot token: missing token

### DISCUSSION THREAD

**Discussion by bosii** [community/community#116409] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

我的copilot无法使用， 我明明在2024年3月份是有订阅年费的copilot服务的，但是现在我的vscode插件无法使用，
我的vscode的插件提示：
[auth] Invalid copilot token: missing token: 403 
[error] [auth] Extension activation failed: "Contact Support. You are currently logged in as

这些问题导致我无法正常使用这个工具

**Comment by Akash1134** [comment]
Hi @bosii , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

➡️  However, to assist you with your Copilot access, I reviewed your account and can see that you need to review/renew your Copilot subscription. If you have already renewed it, please [contact our support team](https://support.github.com/contact) so they can take a look at your account and billing status.
